packages/react/src/global-alert/GlobalAlert.tsx

```diff
@@ -5,11 +5,14 @@ import { Stack } from '../stack';
 import { boxPalette, tokens } from '../core';
 import { WarningFilledIcon, InfoFilledIcon } from '../icon';
 import { Heading } from '../heading';
+import { getOptionalCloseHandler } from '../getCloseHandler';
 import { GlobalAlertCloseButton } from './GlobalAlertCloseButton';
 
 export type GlobalAlertProps = PropsWithChildren<{
-	/** Function to be called when the 'Close' button is pressed. */
+	/** @deprecated use `onClose` instead */
 	onDismiss?: MouseEventHandler<HTMLButtonElement>;
+	/** Function to be called when the 'Close' button is pressed. */
```

Comment:
Probably worth adding this comment to all of the `onClose` props

Reply:
Added